webpack.config.devtools.js

```diff
@@ -28,9 +28,7 @@ module.exports = webpackConfig => {
         return;
       }
       callback();
-    },
-
-    { codemirror: "var devtoolsRequire('devtools/client/sourceeditor/editor')" }
```

Comment:
It was leftover from previous codemirror work; we don't need to map it because we are mapping the `source-editor` file itself which includes codemirror. I noticed it here but I can separate it out if you want.